pkg/proxy/iptables/proxier_test.go

```diff
@@ -708,6 +709,222 @@ func errorf(msg string, rules []iptablestest.Rule, t *testing.T) {
 	t.Errorf("%v", msg)
 }
 
+func TestOverallIPTablesRulesWithMultipleServices(t *testing.T) {
```

Comment:
Can you add a comment explaining what in particular this test checks that isn't checked by one of the other tests already?

Reply:
done. The main difference is that we don't have an overall test that tells us how the NAT rules table looks like.